packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -70,33 +80,46 @@ const ModalContent = React.forwardRef((props, ref) => {
     classNameModal,
     classNameOverlay,
     closeOnOutsideClick,
+    showModalDomUpdates,
+    hideModalDomUpdates,
     onRequestClose,
     role,
     isFullscreen,
     isScrollable,
     zIndex,
     ...customProps
   } = props;
-  let zIndexLayer = '6000';
 
+  React.useEffect(() => {
+    showModalDomUpdates();
+
+    return hideModalDomUpdates;
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  let zIndexLayer = '6000';
   if (zIndexes.indexOf(zIndex) >= 0) {
     zIndexLayer = zIndex;
   }
 
-  const modalClassName = cx(['abstract-modal', {
-    'is-fullscreen': isFullscreen,
-  }, `layer-${zIndexLayer}`, classNameModal]); // Delete the closePortal prop that comes from react-portal.
+  const modalClassName = cx([
+    'abstract-modal',
+    { 'is-fullscreen': isFullscreen },
+    `layer-${zIndexLayer}`,
+    classNameModal,
+  ]);
 
+  // Delete the closePortal prop that comes from react-portal.
```

Comment:
can we also comment on why we remove this prop?

Reply:
@yuderekyu I'm not sure why this was the case. I could dig in if you want me to, but this is just moving the comment from above to be with the delete actions: https://github.com/cerner/terra-framework/pull/1041/files/13b320052eafd45eb4688fe4a142d0f7202aab06#diff-d613e8d30ee292727e9e04177b5d1807L88